ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -85,19 +91,52 @@ StackLayout {
                     }
                 }
             }
+
+            CommunityIntroDialog {
```

Comment:
The `JoinCommunityView` component (`id: joinCommunityViewComponent`) already has this `CommunityIntroDialog`; why duplicate it here?